core/src/main/java/com/netease/arctic/table/TableMetaStore.java

```diff
@@ -117,7 +117,7 @@ public static Builder builder() {
 
   private TableMetaStore(
```

Comment:
Could you log the krb related info before login? Log the file conf would be helpful for trouble-shoot.